crm_lead_lost_reason/README.md

```diff
@@ -0,0 +1,23 @@
+Opportunity Lost Reason
+========================
+
+When an opportunity is lost, a reason must be given,
+it is chosen from a configured list.
+(strongly inspired from sale_cancel_reason addon)
+
+Credits
+=======
+
+Contributors
+------------
+
+* Romain Deheele (<romain.deheele@camptocamp.com>)
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
```

Comment:
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.